<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBoarder</class>
 <widget class="QWidget" name="AddBoarder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>819</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButtonCancel">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>530</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelLastName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>370</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Last Name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAddBoarder">
   <property name="geometry">
    <rect>
     <x>362</x>
     <y>490</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Add Boarder</string>
   </property>
   <property name="shortcut">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditLastName">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelFirstName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>280</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>First Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>C:/Users/GianFrancesSacay/Desktop/RoomRental/RoomRental/images/bbb.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="labelRoomNum">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Room #:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSecondName">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>310</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelHomeAddress">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>400</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Home Address:</string>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>520</y>
     <width>16</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditHomeAddress">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>400</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelMiddleName">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>340</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Middle Name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCourse">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>460</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>141</width>
     <height>26</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Select</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>71</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>72</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>73</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>77</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>99</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="lineEditFirstName">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMiddleName">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSchool">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>430</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelSchool">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>430</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>School:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>90</y>
     <width>271</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>ADD BOARDER</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditBalance">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>490</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelGender">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Gender:</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>190</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="currentSection">
    <enum>QDateTimeEdit::NoSection</enum>
   </property>
   <property name="displayFormat">
    <string>m/d/yyyy</string>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>490</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelDateEntered">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Date Entered:</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Male</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Female</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelBalance">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>490</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Balance:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelSecondNumber">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>310</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Second Name (Optional):</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCourse">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>460</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Course:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
